src/org/usfirst/frc/team5590/robot/subsystems/Grabber.java

```diff
@@ -0,0 +1,81 @@
+package org.usfirst.frc.team5590.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
+//**MUST IMPORT FOR TALONPORT**
+
+/*
+ * 
+ */
+public class Grabber extends Subsystem {
+
+	private static final int SOLENOID_IN = 2;
+	private static final int SOLENOID_OUT = 3;
+	private static final int TALONPORT = 5;
+
+	private static final double SPINNER_SPEED = .7;
+
+	DoubleSolenoid grabberSolenoid;
+	SpeedController spinSpeed;
+
+	public Grabber() {
+		grabberSolenoid = new DoubleSolenoid(SOLENOID_IN, SOLENOID_OUT);
+		// spinSpeed = new TalonSRX(TALONPORT);
```

Comment:
We can instantiate the left and right side speed controllers now, so fix the constructor call here.